composer.lock

```diff
@@ -552,7 +553,7 @@
             },
             "dist": {
                 "type": "zip",
-                "url": "https://api.github.com/repos/elifesciences/api-raml/zipball/6ed114556d3c0d8173d5ce9f164a2e64cf3f2467",
+                "url": "https://api.github.com/repos/elifesciences/api-raml/zipball/c5a072d3006b66545cd395cb57d622d3fd69cd02",
                 "reference": "6ed114556d3c0d8173d5ce9f164a2e64cf3f2467",
```

Comment:
@giorgiosironi I really don't understand this, how can the reference change, but the zipball not!